docs/source/quickstart_mlops.rst

```diff
@@ -0,0 +1,138 @@
+
+Now, your model is available for deployment. You can see it in the **Models** page of the MLflow UI. Open the page for the model you just registered.
+
+You can add a description for the model, add tags, and easily navigate back to the source run that generated this model. You can also transition the model to different stages. For example, you can transition the model to **Staging** to indicate that it is ready for testing. You can transition it to **Production** to indicate that it is ready for deployment.
+
+Transition the model to **Staging** by choosing the **Stage** dropdown:
+
+.. image:: _static/images/quickstart_mlops/mlflow_registry_transitions.png
+    :width: 800px
+    :align: center
+    :alt: Screenshot of MLflow tracking UI models page showing the registered model
+
+Serve the model locally
+----------------------------
+
+MLflow allows you to easily serve models produced by any run or model version. You can serve the model you just registered by running:
+
+.. code-block:: bash
+
+  mlflow models serve -m "models:/wine-quality/Staging" --port 5002
+
+(Note that specifying the port as above may be necessary if you are running the tracking server on the same machine at the default port of **5000**.)
+
+You could also have used a `runs:/<run_id>` URI to serve a model, or any supported URI described in :ref:artifact_stores:.
+
+To test the model, you can send a request to the REST API using the ``curl`` command:
+
+.. code-block:: bash
+
+  curl -d '{"dataframe_split": {"columns": ["fixed acidity","volatile acidity","citric acid","residual sugar","chlorides","free sulfur dioxide","total sulfur dioxide","density","pH","sulphates","alcohol"], "data": [[7,0.27,0.36,20.7,0.045,45,170,1.001,3,0.45,8.8]]}}' -H 'Content-Type: application/json' -X POST localhost:5002/invocations
+
+Inferencing is done with a JSON `POST` request to the **invocations** path on **localhost** at the specified port. The columns specify the names of the columns in the input data. The data is a list of lists, where each inner list is a row of data. For brevity, the above only requests one prediction of wine quality (on a scale of 3-8). The response is a JSON object with a **predictions** key that contains a list of predictions, one for each row of data. In this case, the response is:
+
+.. code-block:: bash
+
+  {"predictions": [{"0": 5.310967445373535}]}%
+
+Build a container image for your model
+---------------------------------------
+
+Most routes toward deployment will use a container to package your model, its dependencies, and much of the runtime environment. You can use MLflow to build a Docker image for your model.
+
+.. code-block:: bash
+
+  mlflow models build-docker --model-uri "models:/quickstart-colorful-colt/1" --name "qs_mlops"
+
+This command builds a Docker image named ``qs_mlops`` that contains your model and its dependencies. It will take several minutes to build the image. Once it completes, you can run the image locally with:
+
```

Comment:
We could also mention that this container can be deployed to any bespoke cloud VM, on-prem, ( or, as mentioned here, locally for testing) to provide real-time inference capabilities.